main/src/ca/uwaterloo/flix/language/dbg/DocAst.scala

```diff
@@ -127,6 +127,12 @@ object DocAst {
 
     case class Native(clazz: Class[_]) extends Atom
 
+    case class StructNew(sym: Symbol.StructSym, exps: List[(Name.Label, Expr)], exp: Expr) extends Composite
+
+    case class StructGet(exp: Expr, field: Name.Label, tpe: Type) extends Composite
```

Comment:
DocAst just have to look right, so get and put probably doesnt have to be case classes.